### DISCUSSION TITLE
Specifiying Tags

### DISCUSSION THREAD

**Discussion by rob-mur** [cucumber-rs/cucumber#125] [Q&A] [answered]
Hi!

Thanks for this repo, I've been very impressed thus far! One question (which may just be me being silly) is how to specify tags when running the tests? I did not see this in the CLI help and it would be very handy to run a single test. If it's just not implemented I can make an issue.

**Comment by tyranron** [comment, marked as answer]
@rob-mur in latest 0.12 `cucumber` release this is possible via `-t, --tags <tagexpr>` CLI option. See ["CLI"](https://cucumber-rs.github.io/cucumber/current/cli.html) and ["Tags"](https://cucumber-rs.github.io/cucumber/current/writing/tags.html) Book chapters for details.
